### PR TITLE
za'lek drone rebalance

### PR DESCRIPTION
**Pull request by Lineth**
Smaller, weaker drones. Medium and large bays. Minor manned Za'lek ship tweaks.

### CODE REVIEW

**Comment by bobbens** [comment]
Some notes before I check it out in detail:
- Would it be possible for you to remove the graphics from this and do the naev-artwork-production or want me to do it for you?
- It seems like you didn't regenerate the collision polygon stuff

File: dat/outfits/launchers/zalek_bomber_drone_fighter_dock.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<outfit name="Za'lek Bomber Drone Fighter Dock">
+ <general>
+  <slot>weapon</slot>
+  <size>large</size>
+  <license>Light Combat Vessel License</license>
+  <rarity>2</rarity>
+  <mass>200</mass>
+  <price>1200000</price>
+  <gfx_store>zalek_drone_bomber_fighter_swarm.png</gfx_store>
```
Comment (bobbens) [inline dat/outfits/launchers/zalek_bomber_drone_fighter_dock.xml:10]: You didn't seem to include this file. If you do it should be pushed to naev-production-artwork instead unless you want me to do it for you.

File: dat/outfits/launchers/zalek_heavy_drone_fighter_dock.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<outfit name="Za'lek Heavy Drone Fighter Dock">
+ <general>
+  <slot>weapon</slot>
+  <size>large</size>
+  <license>Light Combat Vessel License</license>
+  <rarity>2</rarity>
+  <mass>320</mass>
+  <price>1200000</price>
+  <gfx_store>zalek_drone_heavy_fighter_swarm.png</gfx_store>
```
Comment (bobbens) [inline dat/outfits/launchers/zalek_heavy_drone_fighter_dock.xml:10]: Here to.

File: dat/outfits/launchers/zalek_light_drone_fighter_dock.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<outfit name="Za'lek Light Drone Fighter Dock">
+ <general>
+  <slot>weapon</slot>
+  <size>large</size>
+  <license>Light Combat Vessel License</license>
+  <rarity>2</rarity>
+  <mass>240</mass>
+  <price>1200000</price>
+  <gfx_store>zalek_drone_light_fighter_swarm.png</gfx_store>
```
Comment (bobbens) [inline dat/outfits/launchers/zalek_light_drone_fighter_dock.xml:10]: And here.

File: dat/ships/zalek_demon.xml [hunk 4]
```diff
@@ -25,10 +25,12 @@
   <cargo>50</cargo>
  </characteristics>
  <slots>
```
Comment (bobbens) [inline dat/ships/zalek_demon.xml:27]: I have a feeling that this is making it much stronger than it was, but I guess I'll have to run numbers and test. What is the reasoning behind adding two slots and making a medium one large?

**Comment by Lineth** [comment]
Already made a pull request on naev-artwork-production with the images. Just need to change the extensions to webp, yeah?

**Comment by bobbens** [comment]
> Already made a pull request on naev-artwork-production with the images. Just need to change the extensions to webp, yeah?

Yes, that part would be fine.

Regarding the ship graphics themselves, it would be great if you could remove them from here and also include them as webp in the naev-artwork-production PR. Plus you will probably need to regenerate the collision masks if you haven't due to the size change.

File: dat/ships/zalek_demon.xml (hunk 4 above)
Comment (Lineth) [reply dat/ships/zalek_demon.xml:27]: > I have a feeling that this is making it much stronger than it was, but I guess I'll have to run numbers and test. What is the reasoning behind adding two slots and making a medium one large?

My logic was that it's outfits were worse than the Sting's. Will probably revisit it this when I feel confident enough to add the drone bays to the equip file.

**Comment by bobbens** [comment]
I have merged straight into rebalance as I don't want to mess up the outfits in main... yet. Thanks!

**Pull request closed without merging**